lib/sql-manager/index.test.js

```diff
@@ -65,17 +66,20 @@ test.before(async (t) => {
   })
 
   for (const person of Object.values(t.context.people.data)) {
-    await dbm._qb.insert({
-      table: t.context.people.db,
-      data: person
-    })
+    await dbm._qb(t.context.people.db).insert(person)
```

Comment:
Why not use `add`?

Reply:
didn't want the hook to depend on something that is being tested in this unit